src/tools/Reporting/Reporting/Os.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Reporting
+{
+    public class Os
```

Comment:
nit: all other types in this project have full names, so maybe a better name would be `OperatingSystem`?

Reply:
Probably. I worked backward from what we typed in the json (which I just typed out by hand as it's the contract between parts of the system.) I think it isn't worth changing now (since it would change in a bunch of places) but I will keep the feedback in mind for future!